content/en/tools/search.md

```diff
@@ -27,6 +27,7 @@ A static website with a dynamic search function? Yes, Hugo provides an alternati
 * [hugofastsearch](https://gist.github.com/cmod/5410eae147e4318164258742dd053993). A usability and speed update to "GitHub Gist for Fuse.js integration" — global, keyboard-optimized search.
 * [JS & Fuse.js tutorial](https://makewithhugo.com/add-search-to-a-hugo-site/) A simple client-side search solution, using FuseJS (does not require jQuery).
 * [Pagefind](https://github.com/cloudcannon/pagefind). A fully static search library that aims to perform well on large sites, while using as little of your users' bandwidth as possible.
+* [Hugo Lyra](https://github.com/paolomainardi/hugo-lyra). Hugo-Lyra is a javascript module to integrate [Lyra](https://github.com/LyraSearch/lyra) into a Hugo website. It contains the server-side part to generate the index and the client-side library (optional) to bootstrap the search engine easily.
```

Comment:
Looks good! Can you please fix capitalization of 'JavaScript'.

Reply:
@coliff done, thanks!